src/components/MyKiva/BadgeModalContentJourney.vue

```diff
@@ -44,7 +44,7 @@
 							<BadgeContainer
 								:status="getBadgeStatus(index)"
 								:shape="getBadgeShape(badgeWithVisibleTiers.id)"
-								style="width: 133px; height: 133px;"
+								style="width: 128px; height: 128px;"
```

Comment:
Do we also need to adjust the size for the dashed border to be closer to the outside of the badge image?

Reply:
Or do we want the dashed line to be within the badge image now?